app/src/main/java/com/kkolontay/theworld/model/Country.kt

```diff
@@ -1,19 +1,22 @@
 package com.kkolontay.theworld.model
 
+import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 data class CountryName(
-     val common: String)
+    @Json(name = "common") val common: String)
 
 @JsonClass(generateAdapter = true)
 data class CountryFlags(
-     val png: String)
+    @Json(name = "png") val png: String)
 
 @JsonClass(generateAdapter = true)
 data class Country(
-     val name: CountryName,
-    val capital: List<String>?,
-    val population: Long,
-     val area: Double,
-     val flags: CountryFlags)
\ No newline at end of file
+    @Json(name = "name") val name: CountryName,
```

Comment:
this annotation is not needed, it's done automatically. You would only need this if you wanted to make your variable names different from the names in the JSON

Reply:
resolved